sssom/util.py

```diff
@@ -495,10 +494,13 @@ def merge_msdf(
     msdf_with_meta = [inject_metadata_into_df(msdf) for msdf in msdfs]
 
     # merge df [# 'outer' join in pandas == FULL JOIN in SQL]
-    df_merged = reduce(
-        lambda left, right: left.merge(right, how="outer", on=list(left.columns)),
-        [msdf.df for msdf in msdf_with_meta if msdf.df is not None],
-    )
+    # df_merged = reduce(
+    #     lambda left, right: left.merge(right, how="outer", on=list(left.columns)),
+    #     [msdf.df for msdf in msdf_with_meta if msdf.df is not None],
+    # )
+    df_list = [msdf.df for msdf in msdf_with_meta if msdf.df is not None]
+    df_merged = pd.concat(df_list, axis=0, ignore_index=True)
```

Comment:
The main reason I changed it was because it was giving an error stating that you cannot merge a float and object. I think I fixed that issue while working in other places. Reverting this back.